api/src/pcapi/routes/backoffice_v3/docs/JSDOC.md

```diff
@@ -0,0 +1,809 @@
+    data-user-confirmation-modal="false"
+  >
+    Valider
+  </button>
+  <button
+    disabled
+    type="button"
+    class="btn btn-outline-primary"
+    data-url="/backofficev3/pro/validation/user-offerer/batch-pending"
+    data-title="Mettre en attente le(s) rattachement(s)"
+    data-batch-confirm-id="pending-modal"
+    data-modal-button-text="Mettre en attente"
+    data-use-confirmation-modal="true"
+  >
+    Mettre en attente
+  </button>
+  <button
+    disabled
+    type="button"
+    class="btn btn-outline-primary"
+    data-url="/backofficev3/pro/validation/user-offerer/batch-reject"
+    data-title="Rejeter le(s) rattachement(s)"
+    data-batch-confirm-id="rejecting-modal"
+    data-modal-button-text="Rejeter"
+    data-use-confirmation-modal="true"
+  >
+    Rejeter
+  </button>
+</div>
+```
+
+## PcFormsCheckValidity
+
+**Extends PcAddOn**
+
+This edit default `submit` event on forms.
+It internally run first  `form.checkValidity()` and abort in case of invalid form.
+
+> There is no markup necessary as this select all `<form />`
+
+## PcOverrideCustomTextareaEnter
+
+**Extends PcAddOn**
+
+This class trick the default event of a `<textarea />` by submitting the form on <kbd>ENTER</kbd>,
+and go to next line on <kbd>CTRL</kbd>+<kbd>ENTER</kbd> or <kbd>SHIFT</kbd>+<kbd>ENTER</kbd>.
+
+### Examples
+
+```javascript
+<textarea class="pc-override-custom-textarea-enter"></textarea>
+```
+
+## PcPostalAddressAutocomplete
+
+**Extends PcAddOn**
+
+Render a postal address autocomplete that fills value in hidden inputs.
+
+It can be used when declaring form fields in the backend using `fields.PcPostalAddressAutocomplete`.
+
+> There is no HTML markup example as it use a wtforms fields: `postal_address_autocomplete.html`
+
+### Examples
+
+```javascript
+// We declare field mapping to tell what field the autocomplete must set.
+// None of them are mandatory, however, choose one so your autocomplete can submit something to the backend..
+
+postal_address_autocomplete = fields.PcPostalAddressAutocomplete(
+  "Adresse",
+  address="address", // address field name within your form to be autocompleted
+  city="city", // city field name within your form to be autocompleted
+  postalCode="postalCode", // postalCode field name within your form to be autocompleted
+  latitude="latitude", // latitude field name within your form to be autocompleted
+  longitude="longitude", // longitude field name within your form to be autocompleted
+  required=True, // if True, we will add required attribute to autocomplete input (default: false)
+  hasReset=True, // if True, it will add a reset to initial values button (default: false)
+  hasManualEditing=True, // if True, it will propose manual edition (default: false)
+  limit=10, // limit of autocompletes choices (default: 10)
+)
+// Below are the hidden fields, you can decide to add form validation on them.
+address = fields.PCOptHiddenField("address") // if set, it will be set on selection
```

Comment:
Tu veux dire quoi par "it will be set on selection" ?

Reply:
Je veux dire que si tu séléctionne un choix de l'auto complete, le field address sera rempli si déclaré dans ton form